src/domain/event/eventCard/LargeEventCard.tsx

```diff
@@ -30,6 +31,7 @@ interface Props {
 const LargeEventCard: React.FC<Props> = ({ event }) => {
   const { t } = useTranslation();
   const { push } = useHistory();
+  const params = useParams<{ place?: string }>();
```

Comment:
Could be useful to mention here why place is got from the url parameter. It's not 100% clear that place is used only on the place specific paces. Can be little bit confusing to new developer in the team.